getting_started/data_backup.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Data Backup
+layout: default
+parent: Getting Started
+nav_order: 3
+---
+
+# Data Backup
+
+While data storage on Arjuna is [RAID] configured to provide robustness against disk failure, disaster can strike (and did, RIP a lot of things 2021), so it's individual users' responsibility to **back up your data regularly**, _especially_ things that would take a lot of (human and/or computer) time to reproduce! There are a few approaches you can take to do this (outlined on this page), and you will likely want to choose a combination of them for various types of files/code/data. **We strongly recommend that users configure backups to run in a scheduled, automatic fashion** (e.g. using cron jobs) to avoid the need to remember to back up and the possibility of attendant data loss. Instructions to set this up are provided below.
+
+[RAID]: https://en.wikipedia.org/wiki/RAID
+
+## Grabbing a single file: `scp`
+If you just need to move a single file/directory onto/off of Arjuna, the easiest approach is the `scp` command, which has identical syntax to `cp`, but for remote systems requires a full specification of the file's location, for example:
+
+```bash
+# copy a file from arjuna to local machine
+scp username@arjuna.psc.edu:/home/username/src_dir/myfile ~/dest_dir/
```

Comment:
Maybe `scp username@arjuna.psc.edu:~/src_dir/myfile ~/dest_dir` as you're already introducing the concept of `~`.

Or `scp username@arjuna.psc.edu:/path/to/file/on/arjuna/precious.txt .` - To more closely mirror the 2nd command